lib/container/container.go

```diff
@@ -129,3 +131,13 @@ func WaitContainer(id string) (int, error) {
 
 	return client.WaitContainer(id)
 }
+
+func getDockerHost() (host string) {
+	host = Os.Getenv("DOCKER_HOST")
```

Comment:
Change `Os` to `os` to get this to compile. You'll also need to add `"os"` to the import list.